web-app/CKS.Web/Startup.cs

```diff
@@ -42,10 +45,34 @@ public void ConfigureServices(IServiceCollection services)
 			}
 
 			services.AddControllers();
+
+			if (!CurrentEnvironment.IsDevelopment())
+			{
+				services.AddHsts(options =>
```

Comment:
We don't we just always add the services, but we only _use_ them in production? That way we don't need to inject the IWebHostEnvironment  above and it's all a bit simpler?